library/ssl_tls13_keys.c

```diff
@@ -690,7 +690,11 @@ int mbedtls_ssl_tls13_calculate_verify_data( mbedtls_ssl_context* ssl,
     MBEDTLS_SSL_DEBUG_MSG( 2, ( "<= mbedtls_ssl_tls13_calculate_verify_data" ) );
 
 exit:
-
+    /* Erase handshake secrets */
+    if( from == MBEDTLS_SSL_IS_CLIENT )
+        mbedtls_platform_zeroize( base_key, sizeof( ssl->handshake->tls13_hs_secrets.client_handshake_traffic_secret ) );
```

Comment:
The line is too long. Please wrap it. Otherwise, better to always to this thus please move the setting of `base_key` at the beginning of the function just after the local variable declarations and thus before any error can occur. Then ensure that in all error cases we go to `exit`. Only one to change I think line 667:
```C
if( dst_len < md_size )
{
    ret = MBEDTLS_ERR_SSL_BUFFER_TOO_SMALL;
    goto exit;
}
```